pkg/stanza/fileconsumer/attrs/attrs.go

```diff
@@ -5,22 +5,28 @@ package attrs // import "github.com/open-telemetry/opentelemetry-collector-contr
 
 import (
 	"fmt"
+	"os"
+	"os/user"
 	"path/filepath"
 	"runtime"
+	"syscall"
 )
 
 const (
 	LogFileName         = "log.file.name"
 	LogFilePath         = "log.file.path"
 	LogFileNameResolved = "log.file.name_resolved"
 	LogFilePathResolved = "log.file.path_resolved"
+	LogFileOwner        = "log.file.owner"
+	LogFileGroup        = "log.file.group"
 )
 
 type Resolver struct {
 	IncludeFileName         bool `mapstructure:"include_file_name,omitempty"`
 	IncludeFilePath         bool `mapstructure:"include_file_path,omitempty"`
 	IncludeFileNameResolved bool `mapstructure:"include_file_name_resolved,omitempty"`
 	IncludeFilePathResolved bool `mapstructure:"include_file_path_resolved,omitempty"`
+	IncludeFileInfos        bool `mapstructure:"include_file_infos,omitempty"`
```

Comment:
Can we split this into two flags please? We have one flag each for the other attributes so I would prefer if we did here as well. I suggest `include_file_owner` / `include_file_group`.

Reply:
ok no problem i just push the commit which split into two conf :)